simulator/sender.cc

```diff
@@ -42,9 +42,9 @@ Sender::Sender(Config* cfg, size_t socket_thread_num, size_t core_offset,
       inter_frame_delay_(inter_frame_delay),
       ticks_all_(frame_duration_ * ticks_per_usec_ /
                  cfg->Frame().NumTotalSyms()),
-      ticks_wnd1_(200000 /* 200 ms */ * ticks_per_usec_ /
+      ticks_wnd1_((40 * frame_duration_ * ticks_per_usec_) /
```

Comment:
We want to keep this as a constant value for the initial warm up phase because when frame_duration is small, we still want the warm up frame duration to be long enough to make sure Agora can process data in time. This is important especially for larger MIMO size such as 64x16.

Reply:
Ok, the issue I was having was where ticks_wnd1_ can be shorter delay than ticks_wnd2_ when you set frame_duration to a large value (if you are trying to artificially slow down the transmission you cannot get larger than 200ms when slow start is enabled.  

What do you think about adding a 'min' 200ms delay for wnd1_?   ticks_wnd1_ = std::max(200000, (40 * frame_duration_ * ticks_per_usec_))

Reply:
Yeah, this make sense.